src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.SystemIAccessibleWrapper.cs

```diff
@@ -0,0 +1,249 @@
+        /// </summary>
+        private class SystemIAccessibleWrapper : IAccessible
+        {
+            private IAccessible _systemIAccessible;
+
+            public SystemIAccessibleWrapper(IAccessible systemIAccessible)
+            {
+                _systemIAccessible = systemIAccessible;
+            }
+
+            public void accSelect(int flagsSelect, object varChild)
+            {
+                Execute(() => _systemIAccessible.accSelect(flagsSelect, varChild));
+            }
+
+            public void accLocation(out int pxLeft, out int pyTop, out int pcxWidth, out int pcyHeight, object varChild)
+            {
+                int left = 0;
+                int top = 0;
+                int width = 0;
+                int height = 0;
+
+                Execute(() =>
+                {
+                    _systemIAccessible.accLocation(out left, out top, out width, out height, varChild);
+
+                    Debug.WriteLineIf(CompModSwitches.MSAA.TraceInfo, "AccessibleObject.AccLocation: Setting " +
+                        left.ToString(CultureInfo.InvariantCulture) + ", " +
+                        top.ToString(CultureInfo.InvariantCulture) + ", " +
+                        width.ToString(CultureInfo.InvariantCulture) + ", " +
+                        height.ToString(CultureInfo.InvariantCulture));
+                });
+
+                pxLeft = left;
+                pyTop = top;
+                pcxWidth = width;
+                pcyHeight = height;
+            }
+
+            public object accNavigate(int navDir, object varStart)
+            {
+                return GetValue(() => _systemIAccessible.accNavigate(navDir, varStart));
+            }
+
+            public object accHitTest(int xLeft, int yTop)
+            {
+                return GetValue(() => _systemIAccessible.accHitTest(xLeft, yTop));
+            }
+
+            public void accDoDefaultAction(object varChild)
+            {
+                Execute(() => _systemIAccessible.accDoDefaultAction(varChild));
+            }
+
+            public object accParent => GetValue(() => _systemIAccessible.accParent);
+
+            public int accChildCount => GetStructValue(() => _systemIAccessible.accChildCount);
+
+            public object get_accChild(object childID) =>
+                GetValue(() => _systemIAccessible.get_accChild(childID));
+
+            public string get_accName(object childID)
+            {
+                return GetValue(() => _systemIAccessible.get_accName(childID));
+            }
+
+            public void set_accName(object childID, string newName)
+            {
+                Execute(() => _systemIAccessible.set_accName(childID, newName));
+            }
+
+            public string accName
+            {
+                get => GetValue(() => _systemIAccessible.accName);
+                set => Execute(() =>
+                {
+                    _systemIAccessible.accName = value;
+                });
```

Comment:
Any reason not to one-line it?
```suggestion
                set => Execute(() => _systemIAccessible.accName = value);
```

Reply:
Fixed. Thanks!